test/functional/suite_test.go

```diff
@@ -125,6 +129,7 @@ var _ = BeforeSuite(func() {
 			keystoneCRDs,
 			infraCRDs,
 			cinderCRDs,
+			horizonCRD,
```

Comment:
If need another change, can we rename this variable similar to others here, horizonCRDs

Reply:
sure, I will do in the next iteration (when I rebase on top of https://github.com/openstack-k8s-operators/horizon-operator/pull/500)